### ISSUE TITLE
Allow to invert concentration scale

### ISSUE THREAD

**Issue by Bernadetadad** [dms-vep/dms-vep-pipeline-3#72]
@jbloom for pH stability we would actually like to plot escape probability on invert x scale, because the lowest pH is supposed to be the most neutralizing. Also having x on a linear scale would be probably preferable here. 
<img width="630" alt="image" src="https://github.com/dms-vep/dms-vep-pipeline-3/assets/22810662/e6967def-4b58-4ab5-816a-77bae84ecec9">

**Comment by jbloom** [comment]
I think the easiest way to do this for now would be to just use units of `7 - pH` for the pH scale. Can you do that? So in other words, assign a pH of 5.6 a value of `7 - 5.6 = 1.4`. Then to get the linear scale just do [this](https://github.com/dms-vep/dms-vep-pipeline-3/blob/main/test_example/data/antibody_escape_config.yml#L39)

**Comment by Bernadetadad** [comment]
Thanks, will do.

**Comment by Bernadetadad** [comment]
@jbloom would it not be better to do `7.4 - pH` as that's our reference ("no treatment") pH.

**Comment by jbloom** [comment]
The no-treatment doesn't end up on the pH scale, so it may not matter. But yes, you could do that one as well, but it should not make any difference so just choose what you think is most intuitive to explain.

**Comment by jennyahn0** [comment]
@jbloom I think the above only allows to change the scale of the y axis to linear but not x axis

**Comment by jbloom** [comment]
@Bernadetadad @jennyahn0, this will be addressed in version 3.5.0 (which should be finalized soon).

Basically, in `antibody_escape_config.yml` under the `assays` key and `pH_stability` subkey, you will want to add the following:
```
assays:
  ...
  pH_stability:
    ..
    concentration_scale:
      type: linear
    concentration_title: 7.4 minus pH 
```
